advanced/Scripts/database_migration/gravity-db.sh

```diff
@@ -48,6 +48,20 @@ upgrade_gravityDB(){
 		# lists into a single table with a UNIQUE domain constraint
 		echo -e "  ${INFO} Upgrading gravity database from version 3 to 4"
 		sqlite3 "${database}" < "${scriptPath}/3_to_4.sql"
+		version=4
+	fi
+	if [[ "$version" == "4" ]]; then
+		# This migration script upgrades the gravity and list views
+		# implementing necessary changes for per-client blocking
+		echo -e "  ${INFO} Upgrading gravity database from version 4 to 5"
+		sqlite3 "${database}" < "${scriptPath}/4_to_5.sql"
+		version=5
+	fi
+	if [[ "$version" == "5" ]]; then
+		# This migration script upgrades the adlist view
+		# to return an ID used in gravity.sh
+		echo -e "  ${INFO} Upgrading gravity database from version 5 to 6"
```

Comment:
Do we need two migration scripts, since this is all in one PR?

Reply:
One could argue whether we even need 5, seeing as the eventual `release/5.0` -> master will be one PR, also.

I should imagine DL has a sane reason for having it separated

Reply:
The reason for separating them on development is for those who follow the development branch.

Reply:
The reason for separating them was that the two changes came with a temporal distance more than one month in between. Even if this is still a more-or-less isolated feature branch, it seems that enough people got interested in it and we know of at least one user actually using and trying it. Because changes to the migration scripts would have needed them to rebuilt their gravity databases from ground up, I felt this would be enough of an issue to warrant to updates.

Reply:
LGTM